packages/aws-rfdk/lib/deadline/lib/render-queue.ts

```diff
@@ -444,6 +460,25 @@ export class RenderQueue extends RenderQueueBase implements IGrantable {
     tagConstruct(this);
   }
 
+  protected onValidate(): string[] {
```

Comment:
10.1.9 is over 6 months old at this point. It's probably just as good to bump the minimum supported version for the RFDK from Deadline 10.1.9 to 10.1.10 instead.

IIRC, Roman's upcoming work will require bumping it to 10.1.12.

Eliminates complexity here and in the additions to the VersionQuery.

Reply:
This method is actually deprecated according to https://docs.aws.amazon.com/cdk/api/latest/docs/constructs.Construct.html

So probably it's not a good idea to use it.

Reply:
That doc reference points to `@aws-cdk/constructs.Construct` whereas the here we're using `@aws-cdk/core.Construct`. The [`onValidate` is not deprecated](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_core.Construct.html#protected-onwbrvalidate) in this case. That being said, I believe we should probably use the `validate` method instead. The `onValidate` is a wrapper of `validate` that subclasses can use to add pre/post logic to the validation lifecycle.